src/js/api/services/ProductSupplierApi.js

```diff
@@ -5,7 +5,10 @@ import {
 import apiClient from 'utils/apiClient';
 
 export default {
-  getPreferenceTypeOptions: (config) => apiClient.get(PREFERENCE_TYPE_OPTIONS, config),
+  getPreferenceTypeOptions: (config) => apiClient.get(
+    PREFERENCE_TYPE_OPTIONS('?includeMultiple=true&includeNone=true'),
```

Comment:
the params should be in the config. You would need to change this:
```js
const config = {
    signal: controller.signal,
};
```
in the `useProductSupplierFilters.jsx`
to include params:
```js
const config = {
    signal: controller.signal,
    params: {
      includeMultiple: true,
      includeNone: true,
    },
};
```

Reply:
fixed